nodes/test/milvus/test_search_semantic.py

```
@@ -0,0 +1,114 @@
+        'ai': MagicMock(),
+        'ai.common': MagicMock(),
+        'ai.common.schema': MagicMock(),
+        'ai.common.store': MagicMock(DocumentStoreBase=_FakeDocumentStoreBase),
+        'ai.common.config': MagicMock(),
+    },
+):
+    from milvus import Store
+
+
+class _Doc:
+    def __init__(self, score, page_content):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)